hilti/runtime/src/tests/bytes.cc

```diff
@@ -39,21 +39,30 @@ TEST_CASE("at") {
 }
 
 TEST_CASE("construct") {
-    CHECK_EQ(Bytes("123", bytes::Charset::ASCII).str(), "123");
-    CHECK_EQ(Bytes("abc", bytes::Charset::ASCII).str(), "abc");
-    CHECK_EQ(Bytes("abc", bytes::Charset::UTF8).str(), "abc");
-
-    CHECK_EQ(Bytes("\xF0\x9F\x98\x85", bytes::Charset::UTF8).str(), "\xF0\x9F\x98\x85");
-    CHECK_EQ(Bytes("\xc3\x28", bytes::Charset::UTF8, bytes::DecodeErrorStrategy::REPLACE).str(), "\ufffd(");
-    CHECK_EQ(Bytes("\xc3\x28", bytes::Charset::UTF8, bytes::DecodeErrorStrategy::IGNORE).str(), "(");
-    CHECK_THROWS_WITH_AS(Bytes("\xc3\x28", bytes::Charset::UTF8, bytes::DecodeErrorStrategy::STRICT).str(), "illegal UTF8 sequence in string", const RuntimeError&);
-
-    CHECK_EQ(Bytes("\xF0\x9F\x98\x85", bytes::Charset::ASCII, bytes::DecodeErrorStrategy::REPLACE).str(), "????");
-    CHECK_EQ(Bytes("\xF0\x9F\x98\x85", bytes::Charset::ASCII, bytes::DecodeErrorStrategy::IGNORE).str(), "");
-    CHECK_THROWS_WITH_AS(Bytes("\xF0\x9F\x98\x85", bytes::Charset::ASCII, bytes::DecodeErrorStrategy::STRICT).str(), "illegal ASCII character in string", const RuntimeError&);
+    CHECK_EQ(Bytes("123", bytes::Charset(bytes::Charset::ASCII)).str(), "123");
```

Comment:
Left-over double-scoping here, instead of the new `Enum`. Below are some more.

Reply:
Incomplete regexp search 😬. I also fixed the remaining instances in `hilti/runtime/src/tests/to_string.cc`.